package.json

```diff
@@ -62,7 +62,8 @@
         "eslint": "^7.25.0",
         "jest": "^26.6.3",
         "ts-jest": "^26.4.4",
-        "typescript": "^4.1.3"
+        "typescript": "^4.1.3",
+        "wait-port": "^0.2.9"
```

Comment:
New dependency, it seems well maintained, using it to detect when docker port opens after spinning up the spawned process.